optuna/distributions.py

```diff
@@ -131,7 +131,7 @@ def _contains(self, param_value_in_internal_repr: float) -> bool:
         if self.low == self.high:
```

Comment:
Since `self.high` is inclusive now, this `if` branch could be removed I think.

Reply:
You are right! thank you.